apps/android/package.json

```diff
@@ -12,7 +12,8 @@
     "code-style": "fluentui-scripts code-style",
     "depcheck": "fluentui-scripts depcheck",
     "lint": "fluentui-scripts eslint",
-    "start": "react-native start",
+    "start": "react-native start --reset-cache",
+    "start --reset-cache": "react-native start --reset-cache",
```

Comment:
You don't need to include the arguments to the key. With just the key "start" you can run "yarn start --reset-cache", and it will pass the argument to the script (`react-native start`)

Reply:
Also, I think pakage.json doesn't like it when the key for a script has a space? Every other script I've seen has had no spaces